index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  
+    <header class="nav">
+        <a href="" class="m-logo" >My Logo</a>
+         <nav>
+           <ul class="header-link">
+             <li class="link-right">
+                <a href="/someurl" class="inline"><b>Portfolio</b></a>
+                <a href="/someurl" class="inline"><b>About</b></a>
+                <a href="/someurl" class="inline"><b>Contact Us</b></a>
+             </li>
+           </ul>
+        </nav>
+    </header>
+    
+
+
+
+     <!-- header images section -->
+    <section class="abl">
+  
+    </section>
+
+    <!-- Header images section closed here -->
+       <!-- Section 2 started -->
+       <section class="section2">
+        <h2 class="h23">I’m Aziz Glad to see you!</h2>
+         <h4  class="paragraph">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
+  <h5 class="ets" >ETS CONNECT</h5>
+ <ul class="social-menue">
+  <li><img src="images/twitter.PNG" alt="abc"  class="lsls"></li>
+  <li><img src="images/linkedinicon.PNG" alt="abc" class="lsls"></li>
+  <li> <img src="images/medium.PNG" alt="abc" class="lsls"></li>
+  <li> <img src="images/github.PNG" alt="abc" class="lsls"></li>
+  <li><img src="images/angellist.PNG" alt="abc" class="lsls"></li>
+ </ul>
+          
+ </section>
+   <!-- section 2 closed -->
+
+   <section class="abd">
+    <div class="left">
+      
+      <img src="images/a.png" alt="resimage">
+
+    </div>
+
+    <div class="right">        
+      <h2 class="tonic">Tonic</h2>
+      <ul>
+        <li class="canopli">
+           <a href="/someurl" class="sec3rlink31">CANOPY</a>
+            <b class="sec3p">.</b>
+            <a href="/someurl" class="sec3rlink31">Backen Dev</a>
+             <b class="sec3p">.</b>
+           <a href="/someurl" class="sec3rlink31">2015</a>
+         </li>
+      </ul>
+<p class="paragraph1"> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+<ul class="ad">
+  <li class="canopli2">
+       <a href="/someurl" class="sec3rlink32">HTML</a>
+       <a href="/someurl" class="sec3rlink32">CSS</a>
+       <a href="/someurl" class="sec3rlink32">JAVASCRIPT</a>
+   </li>
+</ul>
+<button type="submit" class="bb sec3rbutton sec3rlink4"><span class="d">See Projects</span></button>
+</div>
```

Comment:
Kindly make all the works resemble the Figma design for both desktop and mobile 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/187713574-6d72487c-9d3f-4e3e-aae1-b3789beddc67.png)
![image](https://user-images.githubusercontent.com/73714615/187713702-19f14a16-eb48-4dfa-891e-ac1b23960247.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187713775-db59a9d4-0762-4345-afa6-966c014d32f7.png)
![image](https://user-images.githubusercontent.com/73714615/187713846-58475049-c115-4c20-affd-eabe918d4eb4.png)

Reply:
Kindly, I recommend you to follow the suggestions of the previous reviewer, this will help your project to look more PRO 😉